static/app/components/tabs/index.stories.tsx

```diff
@@ -209,6 +209,22 @@ export default storyBook(Tabs, story => {
           </TabPanels>
         </Tabs>
       </SizingWindow>
+      <br />
+      <p>You can also use the "floating" variant, which is used below</p>
+      <SizingWindow>
+        <Tabs>
+          <TabList variant="floating" hideBorder>
```

Comment:
This could use a Matrix to better show the different values along side each other, and in both orientations